app/controllers/patients_controller.rb

```diff
@@ -249,53 +249,170 @@ def update
+                      'through the Move to Household action. Please refresh.'
+      render(json: { error: error_message }, status: :bad_request) && return
+    end
+
+    # ----- Record Updates -----
+
+    # Update the record
+    updated = current_patient.update(responder_id: new_hoh_id)
+
+    if !updated
+      error_message = 'Move to household action failed: Monitoree was unable to be be updated.'
+      render(json: { error: error_message, status: :forbidden }) && return
     else
-      comment = "User added #{current_patient.first_name} #{current_patient.last_name} to the household. #{new_hoh.first_name} #{new_hoh.last_name}"\
+      # Create history item for new HoH
+      comment = "User added monitoree with ID #{current_patient.id} to a household. #{new_hoh.first_name} #{new_hoh.last_name}"\
                 ' will now be responsible for handling the reporting on their behalf.'
       History.monitoring_change(patient: new_hoh, created_by: current_user.email, comment: comment)
-      comment = "User added monitoree to a household. #{new_hoh.first_name} #{new_hoh.last_name} will now be responsible"\
+
+      # Create history item for current patient being moved to a household
+      comment = "User added monitoree to a household. Monitoree with ID #{new_hoh_id} will now be responsible"\
                 ' for handling the reporting on their behalf.'
+      History.monitoring_change(patient: current_patient, created_by: current_user.email, comment: comment)
     end
-    History.monitoring_change(patient: current_patient, created_by: current_user.email, comment: comment)
+  end
+
+  # Removes a record from a household
+  def remove_from_household
+    current_patient_id = params.permit(:id)[:id]&.to_i
 
+    current_patient = current_user.get_patient(current_patient_id)
     current_user_patients = current_user.patients
 
-    # update_all below does not invoke ActiveRecord callbacks and will not automatically check if this incomming
-    # id exists. Patient.exists?(nil) => false
-    redirect_to(root_url) && return unless Patient.where(purged: false).exists?(new_hoh_id.to_i)
+    # ----- Error Checking -----
 
-    patients_to_update = household_ids + [current_patient_id]
+    # Check to make sure user has access to update this record.
+    unless current_user_patients.exists?(current_patient_id)
+      error_message = 'Remove from household action failed: User does not have permissions to update current monitoree.'
+      render(json: { error: error_message }, status: :forbidden) && return
+    end
 
-    # Make sure all household ids are within jurisdiction
-    redirect_to(root_url) && return if patients_to_update.any? do |patient_id|
-      !current_user_patients.exists?(patient_id)
+    # If the current patients is a HoH, they can't be removed.
+    if current_patient.head_of_household
+      error_message = 'Remove from household  action failed: Monitoree is a head of household. Please refresh.'
```

Comment:
extra space between 'household' and 'action' here

Reply:
Fixed here: d18e3ff